<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitWindow</class>
 <widget class="QDialog" name="LimitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>663</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>各类上限破解</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="save_chkbox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;绝大多数上限破解需要扩充存档文件。该选项扩大临时内存，使临时内存大小大于最终存档文件大小，这样可以允许更大的存档文件大小。此处尽可能开了较大的空间（超过原版的4倍）。理论上，如果扩充存档后各项数据并未超出原版上限，存档文件仍然可以通用。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;绝大多数上限破解需要扩充存档文件。该选项扩大临时内存，使临时内存大小大于最终存档文件大小，这样可以允许更大的存档文件大小。此处尽可能开了较大的空间（超过原版的4倍）。理论上，如果扩充存档后各项数据并未超出原版上限，存档文件仍然可以通用。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>扩充存档临时内存大小为0x100000 （原版为0x36710）</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>原地修改类</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="gen_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;p&gt;修改后，注意修改TimesX.ini中的TotalGeneral。&lt;/p&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;修改后，注意修改TimesX.ini中的TotalGeneral。&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>破解武将上限为1024（原版为512）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="path_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;p&gt;即原版的PATH路径的上限。&lt;/p&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;即原版的PATH路径的上限。&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>破解路径上限为256（原版为128）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="flag_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;破解后理论无上限。在TimesX.ini中，将君主的FLAGNUM设置为Things.ini中对应旗帜的Sequence即可。注：盗贼除外，盗贼的FLAGNUM仍应保持为0。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;破解后理论无上限。在TimesX.ini中，将君主的FLAGNUM设置为Things.ini中对应旗帜的Sequence即可。注：盗贼除外，盗贼的FLAGNUM仍应保持为0。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>破解旗帜上限（原版为29）</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="troop_chkbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>破解队伍上限为</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="troop_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>256</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="troop_label">
          <property name="text">
           <string>（原版为128）</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="king_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>破解君主上限为64（原版为32）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="magic_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>破解武将技上限为256（原版为128）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>新增内存修改类</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="city_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;修改后，注意修改TimesX.ini中的TotalCity。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;修改后，注意修改TimesX.ini中的TotalCity。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>破解城市上限为128（原版为64）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="item_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>破解物品上限为256（原版为128）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="pathpoint_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>破解路径点上限为4096（原版为2048）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="advanced_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>以下为高级设置，一般情况下请跟随默认值，不要随意修改。</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="pathpoint_label1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>（高级）路径点储存区域起始地址为</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="pathpoint_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>578000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="item_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>570000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="item_label1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>（高级）物品储存区域起始地址为</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="city_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>57E000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="city_label1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>（高级）城市临时储存区域起始地址为</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="city_label2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="item_label2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0x</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="pathpoint_label2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0x</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="section_label1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>城市需要0x200大小区域（临时城市指针数组）。
物品需要0x5600大小区域，路径点需要0x5800大小区域。</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="section_chkbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(高级）创建1.05+新EXE区段（不改变文件大小）</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="section_label2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>72</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>注意：本修改器不检查上述地址范围是否在可用区段内，仅会检查新区段是否和既有区段冲突。如您指定的上述地址不在下列区段范围内，请自行使用LordPE等工具检查在对应地址处是否存在已有区段。区段应在0x1000级别上对齐。</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="section_label3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>区段起始地址为0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="section_box1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>570000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="section_label4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>，虚拟大小为0x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="section_box2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>20000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LimitWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LimitWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
